tests/integration/features/dbengine_pg/11-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 2
+commands:
+  # After the operator is installed, we can check the available versions match the expected versions from Version Service
+  - script: >
+      bash -c "diff <(kubectl get dbengine/percona-postgresql-operator -n $NAMESPACE -o yaml | yq -P '.status | sort_keys(..)') <(curl -fsSL https://check-dev.percona.com/versions/v1/pg-operator/$PREVIOUS_PG_OPERATOR_VERSION | yq -P '.versions[] | select(.product == \"pg-operator\") | {\"status\": \"installed\", \"operatorVersion\": .operator, \"availableVersions\": {\"engine\": .matrix.postgresql | to_entries.[] as \$item ireduce({}; .[\$item.key] = {\"imageHash\": \$item.value.imageHash, \"imagePath\": \$item.value.imagePath, \"status\": \$item.value.status}), \"backup\": .matrix.pgbackrest | to_entries.[] as \$item ireduce({}; .[\$item.key] = {\"imageHash\": \$item.value.imageHash, \"imagePath\": \$item.value.imagePath, \"status\": \$item.value.status}), \"proxy\": {\"pgbouncer\": .matrix.pgbouncer | to_entries.[] as \$item ireduce({}; .[\$item.key] = {\"imageHash\": \$item.value.imageHash, \"imagePath\": \$item.value.imagePath, \"status\": \$item.value.status})}}} | sort_keys(..)')"
```

Comment:
😨
This is very hard to read and understand. Can we move it to a separate shell script and call it from here? Or atleast multi-line it? 🙈

Reply:
I know :joy: I thought exactly the same when I was committing this
I'll try to figure out a way of improving this so that it's more readable and maintainable (not that I expect this to change often but still...)

Reply:
@mayankshah1607 please check if 8a9c52e06aa437b8371ae2066479e18630bd5d51 is enough